Editor/Configuration/DefaultEditorConfiguration.cs

```diff
@@ -59,5 +60,16 @@ public virtual AllowedMenuItemsSettings AllowedMenuItemsSettings
             }
             set { allowedMenuItemsSettings = value; }
         }
+
+        internal virtual IValidationHandler Validation { get; }
+
+        protected DefaultEditorConfiguration()
+        {
+#if CREATOR_PRO
+            Validation = new ValidationHandler();
+#else
+            Validation = new NoValidationHandler();
+#endif
```

Comment:
There is a problem: 

If `Innoactive.CreatorEditor.CourseValidation.ValidationHandler` is missing and `CREATOR_PRO` is already defined in the project, `AssemblySymbolChecker` will not even compile to remove the `CREATOR_PRO` symbol.

What if you make `ValidationHandler` inherit from `NoValidationHandler` and then you simply look for (with reflection) the deepest implementation of `IValidationHandler`? 🤔

Reply:
Makes it very prone to breaking if there are at any time more than one type of ValidationHandler. But I cannot come up with a better idea, I think about it.

Reply:
No better idea found, as far we are going to accept the fact that going back from pro does need more than just deleting the folder.